bundles/nightly/pom.xml

```diff
@@ -207,7 +207,7 @@
                         <configuration>
                             <fileSets>
                                 <fileSet>
-                                    <sourceFile>../eclipselink/target/eclipselink.zip</sourceFile>
+                                    <sourceFile>../eclipselink/target/eclipselink-${project.version}.zip</sourceFile>
```

Comment:
I can live with this for now but I'd prefer to this being changed to regular dependencies and using [dependency:copy-dependencies](http://maven.apache.org/plugins/maven-dependency-plugin/copy-dependencies-mojo.html) to gather  what is required in the next PR